src/pages/API.js

```diff
@@ -0,0 +1,68 @@
+export default class API {
```

Comment:
The Api file should be either in src/utils directory or in src/components. Pages directory should contain only index.css and index.js